tests/test_unstructured_storage.py

```diff
@@ -0,0 +1,141 @@
+import re
+import os
+import sys
+import json
+import shutil
+import subprocess
+from time import sleep
+from typing import Generator
+
+import pytest
+from deepdiff import DeepDiff
+from slither import Slither
+from slither.tools.read_storage import SlitherReadStorage
+
+try:
+    from web3 import Web3
+    from web3.contract import Contract
+except ImportError:
+    print("ERROR: in order to use slither-read-storage, you need to install web3")
+    print("$ pip3 install web3 --user\n")
+    sys.exit(-1)
+
+SLITHER_ROOT = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+STORAGE_TEST_ROOT = os.path.join(SLITHER_ROOT, "tests", "storage-layout")
+
+# pylint: disable=too-few-public-methods
+class GanacheInstance:
+    def __init__(self, provider: str, eth_address: str, eth_privkey: str):
+        self.provider = provider
+        self.eth_address = eth_address
+        self.eth_privkey = eth_privkey
+
+
+@pytest.fixture(scope="module", name="web3")
+def fixture_web3(ganache: GanacheInstance):
+    w3 = Web3(Web3.HTTPProvider(ganache.provider, request_kwargs={"timeout": 30}))
+    return w3
+
+
+@pytest.fixture(scope="module", name="ganache")
```

Comment:
You should be able to import these fixtures without reimplementing them

Reply:
I am trying to do this, but when I import them pylint complains that they are unused, which is incorrect since pytest needs them.
```
tests/test_unstructured_storage.py:9:0: W0611: Unused GanacheInstance imported from tests.test_read_storage (unused-import)
tests/test_unstructured_storage.py:9:0: W0611: Unused fixture_ganache imported from tests.test_read_storage (unused-import)
tests/test_unstructured_storage.py:9:0: W0611: Unused fixture_web3 imported from tests.test_read_storage (unused-import)
```

Reply:
The fixtures need to be moved to a `conftest.py`

Reply:
Would that be a new file specific to `test_unstructured_storage`? Or would we be moving the fixtures from `test_read_storage` and importing `conftest.py` into both tests?

Reply:
I read up a little bit on modular pytest fixtures, and it seems my second interpretation is accurate. Would we also need to move all other fixtures from other test programs? Or would it suffice to use `conftest.py` only for `test_read_storage` and `test_unstructured_storage` for now?

Reply:
Assuming the latter is good, this is done.